WebAPI/Controllers/PhotoController.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Core.Common;
+using WebAPI.DTOs;
+using Core.Exceptions;
+using Storage.Models;
+using Core.Interfaces;
+using Core.DTOs;
+using Core.Requests;
+
+namespace WebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PhotoController : ControllerBase
```

Comment:
You can return ServiceResponse directly in controller. Middleware will catch response and will set Status code, so IActionResult is not neccessary. We want alwas the same response structure in our api.